src/languages.ts

```diff
@@ -67,12 +90,17 @@ export const languages: {
             "try",
             "quote",
             "with",
+            "defprotocol",
+            "defimpl",
         ],
         closeTokens: [
             "end",
+            ", do"
```

Comment:
I couldn't make this work with just `do:`, but since all `do:` also require a `,` before, this isn't much of a problem.

Reply:
Make sense 👍